src/cascadia/TerminalApp/TabHeaderControl.idl

```diff
@@ -9,6 +9,7 @@ namespace TerminalApp
     {
         String Title { get; set; };
         Boolean IsPaneZoomed { get; set; };
+        UInt32 MaximumWidth { get; set; };
```

Comment:
nit: should this more specifically be `RenamerMaxWidth`? IDK if in the future we'll want to be able to set the MaximumWidth of the header and have that be different than the renamer, or if the two would ever not be not intrinsically tied to one another

you know what, probably just ignore me on this one

Reply:
We don’t need this property. The value is derived from the max tab width in MUX, and is a reasonable default for all scenarios. Set it once in the xaml and call it a day.

Reply:
Oh, this is how you make it work in different settings scenarios. Not sure I love it... let’s think on it

Reply:
> nit: should this more specifically be RenamerMaxWidth? IDK if in the future we'll want to be able to set the MaximumWidth of the header and have that be different than the renamer, or if the two would ever not be not intrinsically tied to one another

I think that makes a lot of sense, renamed!

> Not sure I love it... let’s think on it

Okay!